packages/amplify-provider-awscloudformation/src/push-resources.js

```diff
@@ -511,6 +511,12 @@ function formNestedStack(context, projectDetails, categoryName, resourceName, se
   /* eslint-enable */
   const initTemplateFilePath = path.join(__dirname, '..', 'resources', 'rootStackTemplate.json');
   const nestedStack = context.amplify.readJsonFile(initTemplateFilePath);
+
+  // Track Amplify Console generated stacks
+  if (process.env.CLI_DEV_INTERNAL_DISABLE_AMPLIFY_APP_DELETION) {
```

Comment:
I would rather set another env var like PROJECT_IS_CONSOLE_APP rather than overloading the deletion env var to also set this description

Reply:
but keep the `CLI_DEV_INTERNAL` prefix, right @edwardfoyle ?

Reply:
@edwardfoyle 100% it'll be 1:1 - we set it on our Tasks running the CLI.

Reply:
I know that, but I meant in the future would there ever be a case where we would want to have DISABLE_AMPLIFY_APP_DELETION = false but also set this value to true?